src/pca.jl

```diff
@@ -148,7 +149,12 @@ function fit{T<:AbstractFloat}(::Type{PCA}, X::DenseMatrix{T};
 
     # delegate to core
     if method == :cov
-        C = Base.covm(X, isempty(mv) ? 0 : mv, 2)
```

Comment:
Likewise here: can you check using `@inferred` and `@code_warntype` that the type is correctly inferred, both inside the function and for its return type?